app/models/work_package/pdf_export/work_package_list_to_pdf.rb

```diff
@@ -109,28 +126,36 @@ def merge_pdfs
   end
 
   def run_batch!(batch_index)
-    initialize_batch_page
+    first = batch_index == 1
 
-    batch_file = render_batch!(batch_index)
+    # We need to clear the page after the first one
+    setup_page!
+
+    batch_file = render_work_packages(work_packages_batch(batch_index),
+                         first: first,
```

Comment:
I know it's nitpicky but the format here is broken. It should either be the whitespace-ladden super indentation or the direct wrap around IMHO! I.e. either

```
batch_file = render_work_packages work_packages_batch(batch_index)
                                  first: first,
                                  filename: "..." do
  write_footers!
end
```

Or my personal preference:

```
batch_file = render_work_packages(
  work_packages_batch(batch_index),
  first: first,
  filename: "..."
) do
  write_footers!
end
```

I dream of the day where formatting is actually completely irrelevant and only an AST is commited.

Reply:
Yes good point, I agree with your style